src/main/java/uk/gov/hmcts/reform/ccd/document/am/configuration/FeignClientConfiguration.java

```diff
@@ -28,9 +41,15 @@ public RequestInterceptor requestInterceptor(FeignHeaderConfig config) {
                         String name = headerNames.nextElement();
                         String value = request.getHeader(name);
                         if (config.getHeaders().contains(name.toLowerCase(Locale.ENGLISH))) {
-                            requestTemplate.header(name, value);
+                            if (name.equals(SERVICE_AUTHORIZATION)) {
```

Comment:
in that case it will handle by framework through the ServiceRequestAuthorizer filter before landing to case document api service and throw throw the 403,Forbidden,Access Denied error.We don't have any control in this case.

{
    "timestamp": "2020-02-26T12:41:49.236+0000",
    "status": 403,
    "error": "Forbidden",
    "message": "Access Denied",
    "path": "/cases/documents/edbdc865-303b-4583-bf5b-573937b5b7da/binary"
}

Reply:
In Case of API gateway it will replace and add the request headers which comes to our micro-service and authenticate successfully.